### PR TITLE
[REVIEW] Returning nan instead of inf when division is by zero

### LINKED ISSUE #495: [BUG] Division by zero returns inf instead of NaN
**Describe the bug**
Blazingsql returns inf instead of nulls when values are divided by zero.

**Steps/Code to reproduce bug**
Run the query below:
```
select o_totalprice, o_totalprice/0 from orders limit 5
```

Current output:
```
   o_totalprice  EXPR$1
0        833.40     inf
1        844.64     inf
2        862.17     inf
3        872.32     inf
4        881.77     inf
```

**Expected behavior**
Expected output:
```
   o_totalprice  EXPR$1
0        833.40     NaN
1        844.64     NaN
2        862.17     NaN
3        872.32     NaN
4        881.77     NaN
```

**Environment overview**
 - Environment location: Bare-metal
 - Method of cuDF install: conda nightly
 - Method of blazingsql install: sources branch-0.13

### PR DESCRIPTION
**Pull request by rommelDB**
Fixes #495

### CODE REVIEW

File: engine/src/Interpreter/interpreter_ops.cuh [hunk 1]
```diff
@@ -447,13 +447,22 @@ private:
...
 				if (is_string_type(right_type_id)) {
 					right_str_view = static_cast<cudf::string_scalar_device_view*>(scalars_right[op_index])->value();
+					right_valid = true;
 				} else {
 					right_value = get_scalar_value<RightType>(scalars_right[op_index]);
+
+					if(oper == operator_type::BLZ_DIV && right_value == 0) //if div by zero = null
+						right_valid = false;
+					else
+						right_valid = true;
```
Comment (jeanp413) [inline engine/src/Interpreter/interpreter_ops.cuh]: Maybe move this logic to be just in one place in line 598
https://github.com/BlazingDB/blazingsql/blob/ad55fb1207838917e356e128513e955047f2a675/engine/src/Interpreter/interpreter_ops.cuh#L597-L599

Comment (rommelDB) [reply engine/src/Interpreter/interpreter_ops.cuh]: @jeanp413 I moved the logic. Please, check it again.

File: engine/src/Interpreter/interpreter_ops.cuh [hunk 2]
```diff
@@ -450,6 +450,7 @@ private:
 			}	else if(right_position == SCALAR_INDEX) {
 				if (is_string_type(right_type_id)) {
 					right_str_view = static_cast<cudf::string_scalar_device_view*>(scalars_right[op_index])->value();
+					right_valid = true;
```
Comment (jeanp413) [inline engine/src/Interpreter/interpreter_ops.cuh]: I think this line is not necessary, `right_valid` is set to true in line 457
```suggestion
```

Comment (rommelDB) [reply engine/src/Interpreter/interpreter_ops.cuh]: Right, removed deprecated change, thanks!

**Comment by kharoc** [comment]
CON NULOS

1 RAL

========================================================
FAILED TESTS
========================================================
                         TestGroup InputType Result                                                                                                                   TestId
0   Bindable Alias                  orc       Fail   TEST_09                                                                                                                
1   Case                            gdf       Fail   TEST_07,TEST_13                                                                                                        
2   Case                            orc       Fail   TEST_07                                                                                                                
3   Case                            parquet   Fail   TEST_07,TEST_13                                                                                                        
4   Case                            psv       Fail   TEST_07,TEST_13                                                                                                        
5   Cast                            orc       Fail   TEST_09                                                                                                                
6   Coalesce                        gdf       Fail   TEST_12,TEST_13                                                                                                        
7   Coalesce                        orc       Fail   TEST_10,TEST_12,TEST_13                                                                                                
8   Coalesce                        parquet   Fail   TEST_12,TEST_13                                                                                                        
9   Coalesce                        psv       Fail   TEST_12,TEST_13                                                                                                        
10  Common Table Expressions        gdf       Fail   TEST_03                                                                                                                
11  Common Table Expressions        orc       Fail   TEST_03                                                                                                                
12  Common Table Expressions        parquet   Fail   TEST_03                                                                                                                
13  Common Table Expressions        psv       Fail   TEST_03                                                                                                                
14  Date                            orc       Fail   TEST_01                                                                                                                
15  Dir                             gdf       Fail   TEST_08                                                                                                                
16  File System Google Storage      parquet   Fail   TEST_01,TEST_04,TEST_06,TEST_07,TEST_10,TEST_13,TEST_14,TEST_15,TEST_16,TEST_17,TEST_18,TEST_19,TEST_20                
17  File System Google Storage      psv       Fail   TEST_01,TEST_04,TEST_06,TEST_07,TEST_10,TEST_13,TEST_14,TEST_15,TEST_16,TEST_17,TEST_18,TEST_19,TEST_20                
18  File System Local               orc       Fail   TEST_08                                                                                                                
19  File System Local               parquet   Fail   TEST_08                                                                                                                
20  File System Local               psv       Fail   TEST_08                                                                                                                
21  File System S3                  parquet   Fail   TEST_01,TEST_04,TEST_06,TEST_07,TEST_10,TEST_13,TEST_14,TEST_15,TEST_16,TEST_17,TEST_18,TEST_19,TEST_20                
22  File System S3                  psv       Fail   TEST_01,TEST_04,TEST_06,TEST_07,TEST_10,TEST_13,TEST_14,TEST_15,TEST_16,TEST_17,TEST_18,TEST_19,TEST_20                
23  Group by without aggregations   gdf       Fail   TEST_05                                                                                                                
24  Group by without aggregations   orc       Fail   TEST_05                                                                                                                
25  Group by without aggregations   parquet   Fail   TEST_05                                                                                                                
26  Group by without aggregations   psv       Fail   TEST_05                                                                                                                
27  Inner join                      gdf       Fail   TEST_02,TEST_05                                                                                                        
28  Inner join                      orc       Fail   TEST_02,TEST_05                                                                                                        
29  Inner join                      parquet   Fail   TEST_02,TEST_05                                                                                                        
30  Inner join                      psv       Fail   TEST_02,TEST_05                                                                                                        
31  Limit                           gdf       Fail   TEST_02                                                                                                                
32  Limit                           orc       Fail   TEST_02,TEST_07                                                                                                        
33  Limit                           parquet   Fail   TEST_02                                                                                                                
34  Limit                           psv       Fail   TEST_02                                                                                                                
35  Nested Queries                  gdf       Fail   TEST_01,TEST_03                                                                                                        
36  Nested Queries                  orc       Fail   TEST_01,TEST_03                                                                                                        
37  Nested Queries                  parquet   Fail   TEST_01,TEST_03                                                                                                        
38  Nested Queries                  psv       Fail   TEST_01,TEST_03                                                                                                        
39  Non-EquiJoin Queries            gdf       Fail   TEST_08                                                                                                                
40  Non-EquiJoin Queries            orc       Fail   TEST_03,TEST_04,TEST_06,TEST_08                                                                                        
41  Non-EquiJoin Queries            parquet   Fail   TEST_08                                                                                                                
42  Non-EquiJoin Queries            psv       Fail   TEST_08                                                                                                                
43  Simple Distribution From Local  gdf       Fail   TEST_06,TEST_10,TEST_36,TEST_46                                                                                        
44  Simple Distribution From Local  orc       Fail   TEST_00,TEST_06,TEST_10,TEST_21,TEST_22,TEST_23,TEST_25,TEST_26,TEST_27,TEST_28,TEST_31,TEST_36,TEST_43,TEST_46,TEST_49
45  Simple Distribution From Local  parquet   Fail   TEST_06,TEST_10,TEST_36,TEST_46                                                                                        
46  Simple Distribution From Local  psv       Fail   TEST_06,TEST_10,TEST_36,TEST_46                                                                                        
47  TPCH Queries                    gdf       Fail   TEST-08,TEST_12,TEST_13                                                                                                
48  TPCH Queries                    orc       Fail   TEST_01,TEST-08,TEST_13,TEST_14                                                                                        
49  TPCH Queries                    parquet   Fail   TEST-08,TEST_12,TEST_13                                                                                                
50  TPCH Queries                    psv       Fail   TEST-08,TEST_12,TEST_13                                                                                                
51  Tables from Pandas              gdf       Fail   TEST_08                                                                                                                
52  Timestamp                       orc       Fail   TEST_01,TEST_02,TEST_03,TEST_07,TEST_10                                                                                
53  Unify Tables                    gdf       Fail   TEST_02                                                                                                                
54  Unify Tables                    orc       Fail   TEST_02                                                                                                                
55  Unify Tables                    parquet   Fail   TEST_02                                                                                                                
56  Unify Tables                    psv       Fail   TEST_02                                                                                                                
57  Wild Card                       gdf       Fail   TEST_08                                                                                                                
58  Wild Card                       orc       Fail   TEST_08                                                                                                                
59  Wild Card                       parquet   Fail   TEST_08                                                                                                                
60  Wild Card                       psv       Fail   TEST_08                                                                                                                
*******************************************************************************
TOTAL SUMMARY for test suite: 
PASSED: 1037/1202
FAILED: 165/1202
CRASH: 0/1202
TOTAL: 1202
MAX DELTA: 150.0
*******************************************************************************
Aggregations without group by:   Start Mem: 411.1875   End Mem: 481.1875   Diff: 70.0
Coalesce:   Start Mem: 481.1875   End Mem: 415.1875   Diff: -66.0
Column Basis:   Start Mem: 415.1875   End Mem: 539.1875   Diff: 124.0
Common Table Expressions:   Start Mem: 539.1875   End Mem: 515.1875   Diff: -24.0
Count without group by:   Start Mem: 515.1875   End Mem: 439.1875   Diff: -76.0
Date:   Start Mem: 439.1875   End Mem: 517.1875   Diff: 78.0
Timestamp:   Start Mem: 517.1875   End Mem: 437.1875   Diff: -80.0
Full outer join:   Start Mem: 437.1875   End Mem: 437.1875   Diff: 0.0
Group by:   Start Mem: 437.1875   End Mem: 481.1875   Diff: 44.0
Group by without aggregations:   Start Mem: 481.1875   End Mem: 473.1875   Diff: -8.0
Inner join:   Start Mem: 473.1875   End Mem: 439.1875   Diff: -34.0
Left outer join:   Start Mem: 439.1875   End Mem: 439.1875   Diff: 0.0
Non-EquiJoin Queries:   Start Mem: 439.1875   End Mem: 491.1875   Diff: 52.0
Nested Queries:   Start Mem: 491.1875   End Mem: 445.1875   Diff: -46.0
Order by:   Start Mem: 445.1875   End Mem: 437.1875   Diff: -8.0
Predicates With Nulls:   Start Mem: 437.1875   End Mem: 481.1875   Diff: 44.0
Simple String:   Start Mem: 481.1875   End Mem: 465.1875   Diff: -16.0
Unary ops:   Start Mem: 669.1875   End Mem: 667.1875   Diff: -2.0
Unify Tables:   Start Mem: 667.1875   End Mem: 667.1875   Diff: 0.0
Union:   Start Mem: 667.1875   End Mem: 571.1875   Diff: -96.0
Limit:   Start Mem: 571.1875   End Mem: 451.1875   Diff: -120.0
Where clause:   Start Mem: 451.1875   End Mem: 443.1875   Diff: -8.0
Bindable Alias:   Start Mem: 443.1875   End Mem: 505.1875   Diff: 62.0
Case:   Start Mem: 505.1875   End Mem: 529.1875   Diff: 24.0
Cast:   Start Mem: 529.1875   End Mem: 465.1875   Diff: -64.0
Concat:   Start Mem: 465.1875   End Mem: 473.1875   Diff: 8.0
Literal:   Start Mem: 473.1875   End Mem: 463.1875   Diff: -10.0
Dir:   Start Mem: 463.1875   End Mem: 613.1875   Diff: 150.0
File System Local:   Start Mem: 613.1875   End Mem: 579.1875   Diff: -34.0
File System S3:   Start Mem: 579.1875   End Mem: 513.1875   Diff: -66.0
File System Google Storage :   Start Mem: 513.1875   End Mem: 577.1875   Diff: 64.0
Like:   Start Mem: 577.1875   End Mem: 623.1875   Diff: 46.0
Substring:   Start Mem: 499.1875   End Mem: 481.1875   Diff: -18.0
Wild Card:   Start Mem: 481.1875   End Mem: 473.1875   Diff: -8.0
TPCH Queries:   Start Mem: 473.1875   End Mem: 523.1875   Diff: 50.0
Round:   Start Mem: 523.1875   End Mem: 467.1875   Diff: -56.0

SIN NULOS

1 RAL

========================================================
FAILED TESTS
========================================================
                         TestGroup InputType Result                                           TestId
0   Bindable Alias                  orc       Fail   TEST_09                                        
1   Date                            orc       Fail   TEST_01                                        
2   Inner join                      gdf       Fail   TEST_02                                        
3   Inner join                      orc       Fail   TEST_02                                        
4   Inner join                      parquet   Fail   TEST_02                                        
5   Inner join                      psv       Fail   TEST_02                                        
6   Nested Queries                  gdf       Fail   TEST_01,TEST_03                                
7   Nested Queries                  orc       Fail   TEST_01,TEST_03                                
8   Nested Queries                  parquet   Fail   TEST_01,TEST_03                                
9   Nested Queries                  psv       Fail   TEST_01,TEST_03                                
10  Simple Distribution From Local  orc       Fail   TEST_22,TEST_43,TEST_46,TEST_49                
11  TPCH Queries                    orc       Fail   TEST_03,TEST_05,TEST-08,TEST_10,TEST_14,TEST_18
12  Tables from Pandas              gdf       Fail   TEST_14                                        
13  Timestamp                       orc       Fail   TEST_02                                        
14  Unify Tables                    gdf       Fail   TEST_02                                        
15  Unify Tables                    orc       Fail   TEST_02                                        
16  Unify Tables                    parquet   Fail   TEST_02                                        
17  Unify Tables                    psv       Fail   TEST_02                                        
*******************************************************************************
TOTAL SUMMARY for test suite: 
PASSED: 1246/1276
FAILED: 30/1276
CRASH: 0/1276
TOTAL: 1276
MAX DELTA: 182.0
*******************************************************************************
Aggregations without group by:   Start Mem: 425.875   End Mem: 487.875   Diff: 62.0
Coalesce:   Start Mem: 487.875   End Mem: 429.875   Diff: -58.0
Column Basis:   Start Mem: 429.875   End Mem: 519.875   Diff: 90.0
Common Table Expressions:   Start Mem: 519.875   End Mem: 447.875   Diff: -72.0
Count without group by:   Start Mem: 447.875   End Mem: 459.875   Diff: 12.0
Date:   Start Mem: 459.875   End Mem: 495.875   Diff: 36.0
Timestamp:   Start Mem: 495.875   End Mem: 451.875   Diff: -44.0
Full outer join:   Start Mem: 451.875   End Mem: 451.875   Diff: 0.0
Group by:   Start Mem: 451.875   End Mem: 429.875   Diff: -22.0
Group by without aggregations:   Start Mem: 429.875   End Mem: 519.875   Diff: 90.0
Inner join:   Start Mem: 519.875   End Mem: 535.875   Diff: 16.0
Left outer join:   Start Mem: 535.875   End Mem: 535.875   Diff: 0.0
Non-EquiJoin Queries:   Start Mem: 535.875   End Mem: 449.875   Diff: -86.0
Nested Queries:   Start Mem: 449.875   End Mem: 521.875   Diff: 72.0
Order by:   Start Mem: 521.875   End Mem: 431.875   Diff: -90.0
Predicates With Nulls:   Start Mem: 431.875   End Mem: 553.875   Diff: 122.0
Simple String:   Start Mem: 553.875   End Mem: 527.875   Diff: -26.0
Unary ops:   Start Mem: 637.875   End Mem: 629.875   Diff: -8.0
Unify Tables:   Start Mem: 629.875   End Mem: 621.875   Diff: -8.0
Union:   Start Mem: 621.875   End Mem: 591.875   Diff: -30.0
Limit:   Start Mem: 591.875   End Mem: 483.875   Diff: -108.0
Where clause:   Start Mem: 483.875   End Mem: 433.875   Diff: -50.0
Bindable Alias:   Start Mem: 433.875   End Mem: 533.875   Diff: 100.0
Boolean:   Start Mem: 533.875   End Mem: 533.875   Diff: 0.0
Case:   Start Mem: 533.875   End Mem: 517.875   Diff: -16.0
Cast:   Start Mem: 517.875   End Mem: 481.875   Diff: -36.0
Concat:   Start Mem: 481.875   End Mem: 487.875   Diff: 6.0
Literal:   Start Mem: 487.875   End Mem: 591.875   Diff: 104.0
Dir:   Start Mem: 591.875   End Mem: 649.875   Diff: 58.0
File System Local:   Start Mem: 649.875   End Mem: 571.875   Diff: -78.0
File System S3:   Start Mem: 571.875   End Mem: 537.875   Diff: -34.0
File System Google Storage :   Start Mem: 537.875   End Mem: 521.875   Diff: -16.0
Like:   Start Mem: 521.875   End Mem: 703.875   Diff: 182.0
Substring:   Start Mem: 579.875   End Mem: 521.875   Diff: -58.0
Wild Card:   Start Mem: 521.875   End Mem: 543.875   Diff: 22.0
TPCH Queries:   Start Mem: 543.875   End Mem: 485.875   Diff: -58.0
Round:   Start Mem: 485.875   End Mem: 489.875   Diff: 4.0


2 Worker's

========================================================
FAILED TESTS
========================================================
                        TestGroup InputType Result                                           TestId
0  Bindable Alias                  orc       Fail   TEST_09                                        
1  Case                            gdf       Fail   TEST_13                                        
2  Date                            orc       Fail   TEST_01                                        
3  Simple Distribution From Local  orc       Fail   TEST_22,TEST_43,TEST_46,TEST_49                
4  TPCH Queries                    orc       Fail   TEST_03,TEST_05,TEST-08,TEST_10,TEST_14,TEST_18
5  Timestamp                       orc       Fail   TEST_02                                        
*******************************************************************************
TOTAL SUMMARY for test suite: 
PASSED: 1496/1510
FAILED: 14/1510
CRASH: 0/1510
TOTAL: 1510
MAX DELTA: 541.0
*******************************************************************************
Aggregations without group by:   Start Mem: 840.875   End Mem: 1381.875   Diff: 541.0
Coalesce:   Start Mem: 1381.875   End Mem: 1317.875   Diff: -64.0
Common Table Expressions:   Start Mem: 1317.875   End Mem: 1269.875   Diff: -48.0
Count without group by:   Start Mem: 1269.875   End Mem: 1283.875   Diff: 14.0
Date:   Start Mem: 1283.875   End Mem: 1319.875   Diff: 36.0
Timestamp:   Start Mem: 1319.875   End Mem: 1337.875   Diff: 18.0
Full outer join:   Start Mem: 1337.875   End Mem: 1313.875   Diff: -24.0
Group by:   Start Mem: 1313.875   End Mem: 1287.875   Diff: -26.0
Group by without aggregations:   Start Mem: 1287.875   End Mem: 1379.875   Diff: 92.0
Inner join:   Start Mem: 1379.875   End Mem: 1305.875   Diff: -74.0
Left outer join:   Start Mem: 1305.875   End Mem: 1275.875   Diff: -30.0
Non-EquiJoin Queries:   Start Mem: 1275.875   End Mem: 1439.875   Diff: 164.0
Nested Queries:   Start Mem: 1439.875   End Mem: 1463.875   Diff: 24.0
Order by:   Start Mem: 1463.875   End Mem: 1413.875   Diff: -50.0
Predicates With Nulls:   Start Mem: 1413.875   End Mem: 1467.875   Diff: 54.0
Simple String:   Start Mem: 1467.875   End Mem: 1367.875   Diff: -100.0
Unary ops:   Start Mem: 1573.875   End Mem: 1567.875   Diff: -6.0
Unify Tables:   Start Mem: 1567.875   End Mem: 1493.875   Diff: -74.0
Union:   Start Mem: 1493.875   End Mem: 1485.875   Diff: -8.0
Limit:   Start Mem: 1485.875   End Mem: 1467.875   Diff: -18.0
Where clause:   Start Mem: 1467.875   End Mem: 1345.875   Diff: -122.0
Bindable Alias:   Start Mem: 1345.875   End Mem: 1293.875   Diff: -52.0
Boolean:   Start Mem: 1293.875   End Mem: 1381.875   Diff: 88.0
Case:   Start Mem: 1381.875   End Mem: 1323.875   Diff: -58.0
Cast:   Start Mem: 1323.875   End Mem: 1319.875   Diff: -4.0
Concat:   Start Mem: 1319.875   End Mem: 1385.875   Diff: 66.0
Literal:   Start Mem: 1385.875   End Mem: 1359.875   Diff: -26.0
Dir:   Start Mem: 1359.875   End Mem: 1573.875   Diff: 214.0
File System Local:   Start Mem: 1573.875   End Mem: 1483.875   Diff: -90.0
File System S3:   Start Mem: 1483.875   End Mem: 1397.875   Diff: -86.0
File System Google Storage :   Start Mem: 1397.875   End Mem: 1451.875   Diff: 54.0
Like:   Start Mem: 1451.875   End Mem: 1543.875   Diff: 92.0
Substring:   Start Mem: 1389.875   End Mem: 1427.875   Diff: 38.0
Wild Card:   Start Mem: 1427.875   End Mem: 1421.875   Diff: -6.0
TPCH Queries:   Start Mem: 1421.875   End Mem: 1467.875   Diff: 46.0
Round:   Start Mem: 1467.875   End Mem: 1439.875   Diff: -28.0

**Comment by kharoc** [comment]
CON NULOS

1 RAL

*******************************************************************************
TOTAL SUMMARY for test suite: 
PASSED: 1051/1202
FAILED: 151/1202
CRASH: 0/1202
TOTAL: 1202
MAX DELTA: 134.0
*******************************************************************************
Aggregations without group by:   Start Mem: 411.1875   End Mem: 479.1875   Diff: 68.0
Coalesce:   Start Mem: 479.1875   End Mem: 413.1875   Diff: -66.0
Column Basis:   Start Mem: 413.1875   End Mem: 537.1875   Diff: 124.0
Common Table Expressions:   Start Mem: 537.1875   End Mem: 527.1875   Diff: -10.0
Count without group by:   Start Mem: 527.1875   End Mem: 471.1875   Diff: -56.0
Date:   Start Mem: 471.1875   End Mem: 465.1875   Diff: -6.0
Timestamp:   Start Mem: 465.1875   End Mem: 537.1875   Diff: 72.0
Full outer join:   Start Mem: 537.1875   End Mem: 537.1875   Diff: 0.0
Group by:   Start Mem: 537.1875   End Mem: 459.1875   Diff: -78.0
Group by without aggregations:   Start Mem: 459.1875   End Mem: 457.1875   Diff: -2.0
Inner join:   Start Mem: 457.1875   End Mem: 449.1875   Diff: -8.0
Left outer join:   Start Mem: 449.1875   End Mem: 449.1875   Diff: 0.0
Non-EquiJoin Queries:   Start Mem: 449.1875   End Mem: 511.1875   Diff: 62.0
Nested Queries:   Start Mem: 511.1875   End Mem: 463.1875   Diff: -48.0
Order by:   Start Mem: 463.1875   End Mem: 455.1875   Diff: -8.0
Predicates With Nulls:   Start Mem: 455.1875   End Mem: 491.1875   Diff: 36.0
Simple String:   Start Mem: 491.1875   End Mem: 473.1875   Diff: -18.0
Unary ops:   Start Mem: 549.1875   End Mem: 547.1875   Diff: -2.0
Unify Tables:   Start Mem: 547.1875   End Mem: 547.1875   Diff: 0.0
Union:   Start Mem: 547.1875   End Mem: 577.1875   Diff: 30.0
Limit:   Start Mem: 577.1875   End Mem: 479.1875   Diff: -98.0
Where clause:   Start Mem: 479.1875   End Mem: 471.1875   Diff: -8.0
Bindable Alias:   Start Mem: 471.1875   End Mem: 531.1875   Diff: 60.0
Case:   Start Mem: 531.1875   End Mem: 459.1875   Diff: -72.0
Cast:   Start Mem: 459.1875   End Mem: 525.1875   Diff: 66.0
Concat:   Start Mem: 525.1875   End Mem: 517.1875   Diff: -8.0
Literal:   Start Mem: 517.1875   End Mem: 507.1875   Diff: -10.0
Dir:   Start Mem: 507.1875   End Mem: 641.1875   Diff: 134.0
File System Local:   Start Mem: 641.1875   End Mem: 493.1875   Diff: -148.0
File System S3:   Start Mem: 493.1875   End Mem: 585.1875   Diff: 92.0
File System Google Storage :   Start Mem: 585.1875   End Mem: 539.1875   Diff: -46.0
Like:   Start Mem: 539.1875   End Mem: 621.1875   Diff: 82.0
Substring:   Start Mem: 595.1875   End Mem: 571.1875   Diff: -24.0
Wild Card:   Start Mem: 571.1875   End Mem: 573.1875   Diff: 2.0
TPCH Queries:   Start Mem: 573.1875   End Mem: 569.1875   Diff: -4.0
Round:   Start Mem: 569.1875   End Mem: 471.1875   Diff: -98.0



SIN NULOS

1 RAL

========================================================
FAILED TESTS
========================================================
                         TestGroup InputType Result                                           TestId
0   Bindable Alias                  orc       Fail   TEST_09                                        
1   Date                            orc       Fail   TEST_01                                        
2   Inner join                      gdf       Fail   TEST_02                                        
3   Inner join                      orc       Fail   TEST_02                                        
4   Inner join                      parquet   Fail   TEST_02                                        
5   Inner join                      psv       Fail   TEST_02                                        
6   Nested Queries                  gdf       Fail   TEST_01,TEST_03                                
7   Nested Queries                  orc       Fail   TEST_01,TEST_03                                
8   Nested Queries                  parquet   Fail   TEST_01,TEST_03                                
9   Nested Queries                  psv       Fail   TEST_01,TEST_03                                
10  Simple Distribution From Local  orc       Fail   TEST_22,TEST_43,TEST_46,TEST_49                
11  TPCH Queries                    orc       Fail   TEST_03,TEST_05,TEST-08,TEST_10,TEST_14,TEST_18
12  Tables from Pandas              gdf       Fail   TEST_14                                        
13  Timestamp                       orc       Fail   TEST_02                                        
14  Unify Tables                    gdf       Fail   TEST_02                                        
15  Unify Tables                    orc       Fail   TEST_02                                        
16  Unify Tables                    parquet   Fail   TEST_02                                        
17  Unify Tables                    psv       Fail   TEST_02                                        
*******************************************************************************
TOTAL SUMMARY for test suite: 
PASSED: 1246/1276
FAILED: 30/1276
CRASH: 0/1276
TOTAL: 1276
MAX DELTA: 144.0
*******************************************************************************
Aggregations without group by:   Start Mem: 425.875   End Mem: 483.875   Diff: 58.0
Coalesce:   Start Mem: 483.875   End Mem: 523.875   Diff: 40.0
Column Basis:   Start Mem: 523.875   End Mem: 483.875   Diff: -40.0
Common Table Expressions:   Start Mem: 483.875   End Mem: 541.875   Diff: 58.0
Count without group by:   Start Mem: 541.875   End Mem: 547.875   Diff: 6.0
Date:   Start Mem: 547.875   End Mem: 481.875   Diff: -66.0
Timestamp:   Start Mem: 481.875   End Mem: 555.875   Diff: 74.0
Full outer join:   Start Mem: 555.875   End Mem: 555.875   Diff: 0.0
Group by:   Start Mem: 555.875   End Mem: 531.875   Diff: -24.0
Group by without aggregations:   Start Mem: 531.875   End Mem: 495.875   Diff: -36.0
Inner join:   Start Mem: 495.875   End Mem: 483.875   Diff: -12.0
Left outer join:   Start Mem: 483.875   End Mem: 483.875   Diff: 0.0
Non-EquiJoin Queries:   Start Mem: 483.875   End Mem: 537.875   Diff: 54.0
Nested Queries:   Start Mem: 537.875   End Mem: 487.875   Diff: -50.0
Order by:   Start Mem: 487.875   End Mem: 517.875   Diff: 30.0
Predicates With Nulls:   Start Mem: 517.875   End Mem: 507.875   Diff: -10.0
Simple String:   Start Mem: 507.875   End Mem: 471.875   Diff: -36.0
Unary ops:   Start Mem: 669.875   End Mem: 663.875   Diff: -6.0
Unify Tables:   Start Mem: 663.875   End Mem: 659.875   Diff: -4.0
Union:   Start Mem: 659.875   End Mem: 639.875   Diff: -20.0
Limit:   Start Mem: 639.875   End Mem: 533.875   Diff: -106.0
Where clause:   Start Mem: 533.875   End Mem: 463.875   Diff: -70.0
Bindable Alias:   Start Mem: 463.875   End Mem: 553.875   Diff: 90.0
Boolean:   Start Mem: 553.875   End Mem: 435.875   Diff: -118.0
Case:   Start Mem: 435.875   End Mem: 463.875   Diff: 28.0
Cast:   Start Mem: 463.875   End Mem: 437.875   Diff: -26.0
Concat:   Start Mem: 437.875   End Mem: 563.875   Diff: 126.0
Literal:   Start Mem: 563.875   End Mem: 535.875   Diff: -28.0
Dir:   Start Mem: 535.875   End Mem: 679.875   Diff: 144.0
File System Local:   Start Mem: 679.875   End Mem: 623.875   Diff: -56.0
File System S3:   Start Mem: 623.875   End Mem: 557.875   Diff: -66.0
File System Google Storage :   Start Mem: 557.875   End Mem: 619.875   Diff: 62.0
Like:   Start Mem: 619.875   End Mem: 661.875   Diff: 42.0
Substring:   Start Mem: 563.875   End Mem: 535.875   Diff: -28.0
Wild Card:   Start Mem: 535.875   End Mem: 535.875   Diff: 0.0
TPCH Queries:   Start Mem: 535.875   End Mem: 593.875   Diff: 58.0
Round:   Start Mem: 593.875   End Mem: 597.875   Diff: 4.0

2 WORKER's

========================================================
FAILED TESTS
========================================================
                        TestGroup InputType Result                                           TestId
0  Bindable Alias                  orc       Fail   TEST_09                                        
1  Date                            orc       Fail   TEST_01                                        
2  Simple Distribution From Local  orc       Fail   TEST_22,TEST_43,TEST_46,TEST_49                
3  TPCH Queries                    orc       Fail   TEST_03,TEST_05,TEST-08,TEST_10,TEST_14,TEST_18
4  Timestamp                       orc       Fail   TEST_02                                        
*******************************************************************************
TOTAL SUMMARY for test suite: 
PASSED: 1497/1510
FAILED: 13/1510
CRASH: 0/1510
TOTAL: 1510
MAX DELTA: 503.0
*******************************************************************************
Aggregations without group by:   Start Mem: 840.875   End Mem: 1343.875   Diff: 503.0
Coalesce:   Start Mem: 1343.875   End Mem: 1277.875   Diff: -66.0
Common Table Expressions:   Start Mem: 1277.875   End Mem: 1363.875   Diff: 86.0
Count without group by:   Start Mem: 1363.875   End Mem: 1369.875   Diff: 6.0
Date:   Start Mem: 1369.875   End Mem: 1325.875   Diff: -44.0
Timestamp:   Start Mem: 1325.875   End Mem: 1345.875   Diff: 20.0
Full outer join:   Start Mem: 1345.875   End Mem: 1315.875   Diff: -30.0
Group by:   Start Mem: 1315.875   End Mem: 1289.875   Diff: -26.0
Group by without aggregations:   Start Mem: 1289.875   End Mem: 1261.875   Diff: -28.0
Inner join:   Start Mem: 1261.875   End Mem: 1325.875   Diff: 64.0
Left outer join:   Start Mem: 1325.875   End Mem: 1297.875   Diff: -28.0
Non-EquiJoin Queries:   Start Mem: 1297.875   End Mem: 1335.875   Diff: 38.0
Nested Queries:   Start Mem: 1335.875   End Mem: 1315.875   Diff: -20.0
Order by:   Start Mem: 1315.875   End Mem: 1309.875   Diff: -6.0
Predicates With Nulls:   Start Mem: 1309.875   End Mem: 1279.875   Diff: -30.0
Simple String:   Start Mem: 1279.875   End Mem: 1271.875   Diff: -8.0
Unary ops:   Start Mem: 1473.875   End Mem: 1441.875   Diff: -32.0
Unify Tables:   Start Mem: 1441.875   End Mem: 1423.875   Diff: -18.0
Union:   Start Mem: 1423.875   End Mem: 1391.875   Diff: -32.0
Limit:   Start Mem: 1391.875   End Mem: 1339.875   Diff: -52.0
Where clause:   Start Mem: 1339.875   End Mem: 1337.875   Diff: -2.0
Bindable Alias:   Start Mem: 1337.875   End Mem: 1365.875   Diff: 28.0
Boolean:   Start Mem: 1365.875   End Mem: 1287.875   Diff: -78.0
Case:   Start Mem: 1287.875   End Mem: 1279.875   Diff: -8.0
Cast:   Start Mem: 1279.875   End Mem: 1361.875   Diff: 82.0
Concat:   Start Mem: 1361.875   End Mem: 1469.875   Diff: 108.0
Literal:   Start Mem: 1469.875   End Mem: 1433.875   Diff: -36.0
Dir:   Start Mem: 1433.875   End Mem: 1513.875   Diff: 80.0
File System Local:   Start Mem: 1513.875   End Mem: 1455.875   Diff: -58.0
File System S3:   Start Mem: 1455.875   End Mem: 1501.875   Diff: 46.0
File System Google Storage :   Start Mem: 1501.875   End Mem: 1417.875   Diff: -84.0
Like:   Start Mem: 1417.875   End Mem: 1511.875   Diff: 94.0
Substring:   Start Mem: 1451.875   End Mem: 1489.875   Diff: 38.0
Wild Card:   Start Mem: 1489.875   End Mem: 1403.875   Diff: -86.0
TPCH Queries:   Start Mem: 1403.875   End Mem: 1439.875   Diff: 36.0
Round:   Start Mem: 1439.875   End Mem: 1441.875   Diff: 2.0

**Pull request merged**